src/Database/EloquentBuilder.php

```diff
@@ -3,6 +3,7 @@
 namespace Weebly\Mutate\Database;
 
 use Closure;
+use Illuminate\Contracts\Support\Arrayable;
```

Comment:
`use` statement got added, but this doesn't seem to be used anywhere in the file

Reply:
It's actually used here
https://github.com/Weebly/laravel-mutate/blob/aab32b1d37919f76aefb30337e171e7da666fd9b/src/Database/EloquentBuilder.php#L69-L72

This is technically a bugfix ;)